store/src/main/java/org/apache/rocketmq/store/CommitLog.java

```diff
@@ -906,7 +906,7 @@ public CompletableFuture<PutMessageResult> asyncPutMessage(final MessageExtBroke
         }
         // Set the message body CRC (consider the most appropriate setting on the client)
         msg.setBodyCRC(UtilAll.crc32(msg.getBody()));
-        if (enabledAppendPropCRC) {
+        if (!enabledAppendPropCRC) {
```

Comment:
```
 // Set the message body CRC (consider the most appropriate setting on the client)
        msg.setBodyCRC(UtilAll.crc32(msg.getBody()));
        if (!enabledAppendPropCRC) {
```

if !enabledAppendPropCRC ,why should we invoke UtilAll.crc32(msg.getBody())?

Reply:
__CRC32#This property is reserved by the system. The system only needs to add or delete it when enabledAppendPropCRC is true.